src/main/resources/assets/warpdrive/lua.OpenComputers/warpdriveShipController/startup

```diff
@@ -475,7 +475,7 @@ function ship_page_controls()
   
   w.setCursorPos(1, 20)
   w.setColorControl()
-  w.writeFullLine(" set ship Name (N), dImensions (I), Movement (M), Target (T)")
+  w.writeFullLine(" set ship Name (N), dImensions (I), Movement (A/R)")
```

Comment:
Players try to move without exiting the computer. Movement keys defaults to ASWD or QSZD depending on their keyboard layout.
As such, the 'A' key can't be used in menus since players may use those keys involuntarily. Same goes with E for opening inventories.
Sorry I forgot about that in my previous comment.

I propose 'Movement (M/P)' with M for relative (existing), and P for position (absolute).

Reply:
Done.